### PR TITLE
Maint/diy update

### LINKED ISSUE #366: [maint] Point DIY to VM Guide in Community

### PR DESCRIPTION
**Pull request by kn0wmad**
Closes #366

### CODE REVIEW

**Review by gStart9 — APPROVED** [review]

**Comment by gStart9** [comment]
It all looks good to me.  I changed StartOS back to embassyOS for now (thank goodness!)  and after speaking with Aiden I changed the text occurrences of x86 to x86_64 since they are actually different architectures.

File: site/source/developer-docs/packaging.rst [hunk 1]
```diff
@@ -55,7 +55,7 @@ A basic development and testing environment includes:
     
     - Use your own hardware to `DIY <https://start9.com/latest/diy>`_
     - Purchase a device from the `Start9 Store <https://store.start9.com>`_
-    - x86/VM support coming soon
+    - x86_64/VM support coming soon
```
Comment (kn0wmad) [inline site/source/developer-docs/packaging.rst:58]:
```suggestion
    - Run in a Virtual Machine
```

File: site/source/diy/index.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ===
 DIY
 ===
-Here you will find guides for building your own Embassy!  Currently, embassyOS supports the Raspberry Pi and on most x86 platforms (desktops, laptops, mini PCs, servers, and virtual machines).
+Here you will find guides for building your own Embassy!  Currently, embassyOS supports the Raspberry Pi and on most x86_64 platforms (desktops, laptops, mini PCs, servers, and virtual machines).
```
Comment (kn0wmad) [inline site/source/diy/index.rst:6]:
```suggestion
Here you will find guides for building your own Server!  Currently, embassyOS supports the Raspberry Pi and on most x86_64 platforms (desktops, laptops, mini PCs, servers, and virtual machines).
```

**Review by kn0wmad — COMMENTED** [review]
minor edits

**Pull request merged**